src/extractors/custom/otrs.com/index.js

```diff
@@ -0,0 +1,43 @@
+export const OtrsComExtractor = {
+  domain: 'otrs.com',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
+  },
+
+  author: {
+    selectors: ['div.dateplusauthor a'],
+  },
+
+  date_published: {
+    selectors: [['meta[name="article:published_time"]', 'value']],
+  },
+
+  dek: {
+    selectors: [['meta[name="og:description"]', 'value']],
+  },
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['#main article'],
+
+    defaultCleaner: false,
+
+    transforms: {
+      'div.blockquote-inner': $node => {
+        const text = $node.text();
+        $node.html(`<strong>"${text}"</strong>`);
```

Comment:
This feels more like a style change than a behavioral one. What's the reasoning behind wrapping the quoted text in a `<strong>` tag?

Reply:
That's because the original article is like that.

Reply:
Could you please provide some examples? Looking through the HTML fixture, the original blockquotes looks like:

```html
  <blockquote class="gr-14 push-5" style="margin:0px;">

    <div class="blockquote-inner">Working, social contact and an active life helps to maintain performance and supports fitness in older age.&#x201D;</div>
    <cite></cite>
  
  </blockquote>
```

Which also raises another issue with the transform and wrapping the text in double quotes, since it seems like they're manually adding that in some occurrences even though they are also added using CSS as pseudo elements. I am personally in favor of keeping the text intact.

Reply:
Thank you. I decided to keep the text intact.